test/utils/widget_pump.dart

```diff
@@ -16,10 +17,11 @@ Future<void> pumpProviderScoped(WidgetTester tester, Widget widget, [List<Overri
     tester.pumpWidget(
       ProviderScope(
         overrides: overrides,
-        child: MaterialApp(
-          home: Scaffold(
-            body: widget,
-          ),
-        ),
+        child: Builder(builder: (context) {
+          return MaterialApp(
+            theme: ThemeController().currentThemeData(context),
```

Comment:
you should probably use the `themeController` provider

Reply:
By using `Consumer`, right?